CHANGELOG/v1.8.0-beta.1.md

```diff
@@ -0,0 +1,418 @@
+- ClusterCacheTracker: Use indexes field instead of passing it around (#10606)
+- ClusterClass: Add Cluster.GetClassKey() to retrieve a NamespacedName for classes (#10703)
+- ClusterClass: Add MachinePools to autoscaler e2e test (#10083)
+- ClusterClass: Add MachinePools to handler and topology test (#10216)
+- ClusterClass: Deprecate IP Family builtin variable (#10554)
+- ClusterClass: Log diffs for Cluster topology rollouts/patches (#10690)
+- ClusterClass: Separate schema and CEL errors in CC variable validation (#10809)
+- ClusterClass: Webhooks: make MDT.replicas and autoscaler mut. exclusive (#10370)
+- clusterctl: Add ionoscloud provider (#10767)
+- clusterctl: Add support for the linode-linode infrastructure provider to clusterctl (#10471)
+- clusterctl: Add Tinkerbell to the providers list (#10487)
+- clusterctl: Always run crd migration if possible to reduce conversion webhook usage (#10513)
+- clusterctl: Bump cert-manager to 1.14.5 (#10514)
+- clusterctl: Bump cert-manager to v1.15.0 (#10727)
+- clusterctl: Bump cert-manager to v1.15.1 (#10807)
+- clusterctl: Improve cert-manager shouldUpgrade (#10407)
+- ClusterResourceSet: Ensure CRS controller always add ownerReference to resources (#10756)
+- ClusterResourceSet: Make ClusterResourceSet controller more predictable (#10869)
+- Community meeting: Retire the Alternative communication pattern feature group (#10658)
+- Control-plane: Improve KCP remediation re-entrancy (#10559)
+- Core: Cleanup separate unstructuredCachingClient (#10692)
+- Dependency: Bump controller-gen to v0.15 (#10380)
+- Dependency: Bump controller-runtime to v0.18.0 (#10383)
+- Dependency: Bump conversion-gen to v0.30.0 (#10474)
+- Dependency: Bump docker/docker to 26.0.0+incompatible (#10335)
+- Dependency: Bump envtest to v1.30.0 (#10477)
+- Dependency: Bump github.com/distribution/reference from 0.5.0 to 0.6.0 (#10501)
+- Dependency: Bump github.com/docker/docker from 26.1.4+incompatible to 27.0.0+incompatible in /test (#10775)
+- Dependency: Bump github.com/docker/docker to v27.0.1 (#10794)
+- Dependency: Bump Go to v1.22.2 (#10452)
+- Dependency: Bump go version to 1.22.3 (#10586)
+- Dependency: Bump golang to v1.21.9 and golang.org/x/net to mitigate CVE-2023-45288 (#10375)
+- Dependency: Bump golangci-lint to v1.57.2 (#10398)
+- Dependency: Bump to Go 1.22.4 (#10739)
+- Dependency: Bump to Go 1.22.5 (#10828)
+- Dependency: Fix loopvar linter issue and usage of deprecated grpc function (#10731)
+- Dependency: Fixup patch order in CABPK (#10399)
+- Dependency: Stop bumping cel-go via dependabot (#10834)
+- Dependency: Update Kustomize deprecated syntax (#10294)
+- Devtools: Add triage-party for the Cluster API backlog (#10437)
+- Devtools: Bump Cluster API Visualizer to v1.3.0 (#10386)
+- Devtools: Bump Cluster API Visualizer to v1.3.1 (#10816)
+- Devtools: Improve triage query (#10644)
+- Devtools: Tilt ensure in-tree providers always use start.sh to allow restarts (#10811)
```

Comment:
Honestly, dropping all those colons just makes it had to read. Can we please keep them?

Reply:
Addressed

Reply:
Thank you!